packages/tools/src/types/ContourAnnotation.ts

```diff
@@ -7,6 +7,13 @@ export type ContourAnnotationData = {
       polyline: Types.Point3[];
       closed: boolean;
     };
+    handles?: {
+      /**
+       * An index of the handle to contour points.  This allows associating
+       * the handle data to the polyline position data.
+       */
+      handleContourIndex?: number[];
```

Comment:
@lscoderwhat do you think about this?

Reply:
Where is it being used?

Reply:
Removing momentarily - I'm not using it right now.